webapp/src/views/logged-in-structure.vue

```diff
@@ -103,14 +103,6 @@ export default {
       this.$bvModal.show('donate');
     }
   },
-  async created() {
-    if (Auth.isAuthenticated()) {
-      await this.getCurrentUser();
-      await this.refreshData();
-    }
-    else
-      this.$router.push({ name: DEFAULT_SIGNED_OUT_PAGE });
-  },
```

Comment:
Why did you remove this here? Are you sure it doesn't break anything else? @BambanzaJuniorThe2nd